py/examples/tall_stats.py

```diff
@@ -0,0 +1,17 @@
+# Tall stat
+# Create a card with stats laid out vertically 
+# ---
+from h2o_wave import main, app, Q, ui
+
+@app('/demo')
+async def serve(q: Q):
+    # Modify the page
+
+    q.page['example'] = ui.tall_stats_card(box='2 2 4 4', items=[
```

Comment:
Start from the first row and col in order to look good in Tour app. Also since this card is intended to be vertical, the square may not be the best solution here.
```suggestion
    q.page['example'] = ui.tall_stats_card(box='1 1 2 4', items=[
```